conn.go

```diff
@@ -347,6 +376,97 @@ func (conn *Conn) inWorker() {
 	}
 }
 
+// sigWorker runs in an own goroutine, getting signals and handling
```

Comment:
Ok, want me fix inWorker and outWorker docs at the same time? (I based sigWorker's docs on theirs.)